### DISCUSSION TITLE
What is the longest river that flows only in one country?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1156] [Galaxy Trivia] [answered]
**Bonus:** The red color in the 🇳🇵 flag is associated with which flower?

**Comment by tomerf** [comment, marked as answer]
The Yangtze River is the world’s third-longest river and the world’s longest river that runs fully within a single country. It is also Asia’s longest river. One-third of the population of the world’s most populous country, China, lives in the Yangtze River basin.

In Nepal's flag, the crimson red is the symbol of bravery and it also represents the color of the rhododendron, Nepal's national flower.
